Scala/Symbols-type.tmPreferences

```diff
@@ -9,10 +9,6 @@
   <dict>
     <key>showInSymbolList</key>
     <integer>1</integer>
-    <key>symbolTransformation</key>
-    <string>
-      s/(.+)/  type $1/
-    </string>
```

Comment:
This file can be removed since `entity.name.type` is included in the symbol list by default